cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -28,12 +28,17 @@ spec:
             memory: 50Mi
         command:
         - /usr/local/bin/etcd
+        - -data-dir
+        - /var/etcd/data
         - -listen-client-urls
         - http://127.0.0.1:2379,http://127.0.0.1:4001
         - -advertise-client-urls
         - http://127.0.0.1:2379,http://127.0.0.1:4001
         - -initial-cluster-token
         - skydns-etcd
+        volumeMounts:
```

Comment:
I was just asking for -debug https://github.com/coreos/etcd/blob/master/Documentation/configuration.md, but it looks like it isn't available in etcd 2.0 :\